frontend/packages/console-shared/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -110,9 +110,13 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
                           <Label>{itemType}</Label>
                         </SplitItem>
                         <SplitItem>
-                          <TextContent>
-                            <Text component={TextVariants.small}>{item.provider}</Text>
-                          </TextContent>
+                          {item?.attributes?.categories?.length > 0 ? (
+                            <Label color="blue">{item?.attributes?.categories[0]}</Label>
+                          ) : (
```

Comment:
nit:
```suggestion
                          {item.attributes?.categories?.length > 0 ? (
                            <Label color="blue">{item.attributes.categories[0]}</Label>
                          ) : (
```

and we need to check for `categories[0]`?

Reply:
Its just a string array, so the length checks should be enough.